src/js/step.js

```diff
@@ -403,6 +406,10 @@ export class Step extends Evented {
       this.tour._setupModal();
     }
 
+    if (Shepherd.activeTour != this.tour) {
+      this.tour._setupActiveTour()
```

Comment:
Do we need all the setup or can we just `this.tour.modal.show()`?

Reply:
im not 100% sure since it's been a while since i've wrote this but i the main reason to use `_setupActiveTour` here instead of just `show`  was to make sure the correct tour is flagged as active and the 'activation' events are triggered.